.github/workflows/stale_issue.yml

```diff
@@ -0,0 +1,20 @@
+name: "Close stale pull requests/issues"
+on:
+  schedule:
+  - cron: "16 00 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This pull request has been marked as stale because it has been open (more than) 60 days with no activity. Remove the stale label or add a comment saying that you would like to have the label removed otherwise this pull request will automatically be closed in 14 days. Note, that you can always re-open a closed pull request at any time.'
+        stale-issue-message: 'This issue has been marked as stale because it has been open (more than) 60 days with no activity. Remove the stale label or add a comment saying that you would like to have the label removed otherwise this pull request will automatically be closed in 14 days. Note, that you can always re-open a closed pull request at any time.'
+        days-before-stale: 60
+        days-before-close: 14
+        stale-issue-label: 'Stale'
+        stale-pr-label: 'Stale PR'
+        exempt-pr-labels: 'DNM,In progress'
+        exempt-issue-labels: 'In progress,Waiting for response'
```

Comment:
I don't think PR/Issues with "Waiting for response" should be exempt (personally I think this is an 
aggravating circumstance)

Reply:
We have a documented process for "waiting for response" of closing the issue after two requests for response in a two week period...